src/SessionClient.js

```diff
@@ -180,7 +180,7 @@ const prepareParameters = function(startDate, endDate, datasetName = '', targetC
     };
 
     if (datasetName.length > 0) {
-        Object.defineProperty(parameters, 'datasetName', {
+        Object.defineProperty(parameters, 'dataSourceName', {
             value: datasetName,
```

Comment:
Yep, agree.  It was a miss from when we made the change to dataSourceName.  As I get time I'll circle back and update it.